### PR TITLE
SDA-5813 Remove DisplayName from cluster

### PR DESCRIPTION
**Pull request by hunterkepley**
https://issues.redhat.com/browse/SDA-5813

### CODE REVIEW

File: pkg/ocm/clusters.go [hunk 1]
```diff
@@ -264,7 +264,7 @@ func (c *Client) GetClusterByID(clusterKey string, creator *aws.Creator) (*cmv1.
 }
 
 func (c *Client) GetClusterUsingSubscription(clusterKey string, creator *aws.Creator) (*amv1.Subscription, error) {
-	query := fmt.Sprintf("plan.id = 'MOA' AND (display_name  = '%s' OR "+
+	query := fmt.Sprintf("plan.id = 'MOA' AND (name = '%s' OR "+
```
Comment (pvasant) [inline pkg/ocm/clusters.go:267]: Can you check if we have name in the subscription? i think we cant remove this

Comment (hunterkepley) [reply pkg/ocm/clusters.go:267]: Removing change

**Comment by tzvatot** [comment]
lgtm

**Comment by mjlshen** [comment]
fyi this is causing build failures, e.g.: https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_rosa/798/pull-ci-openshift-rosa-master-images-images/1552353397569490944

**Comment by hunterkepley** [comment]
> fyi this is causing build failures, e.g.: https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_rosa/798/pull-ci-openshift-rosa-master-images-images/1552353397569490944

@mjlshen I am going to update the SDK version in ROSA to fix this issue.

**Comment by hunterkepley** [comment]
@mjlshen https://github.com/openshift/rosa/pull/801

**Pull request merged**